src/MovieCard/MovieCard.js

```diff
@@ -6,9 +6,9 @@ import {Link} from 'react-router-dom'
 
 const MovieCard = (props) => {
   const altText = `${props.movie.title} movie poster`
-    const showMoviePage = () => {
-      props.showMoviePage(props.movie.id);
-    }; 
+    // const showMoviePage = () => {
+    //   props.showMoviePage(props.movie.id);
```

Comment:
why'd you comment this out?

Reply:
I didn't think it was being used - but maybe this is what is throwing my error!